index.js

```diff
@@ -1,7 +1,10 @@
 const DD = require('node-dogstatsd').StatsD
 
 module.exports = function (options) {
-  const datadog = options.dogstatsd || new DD()
+  const host = options.host || 'localhost'
+  const port = options.port || 8125
+  const datadog = new DD(host, port)
```

Comment:
Can we make this also accept an existing datadog instance? So:

``` js
const datadog = options.dogstatsd || new DD()
```

Then all is well and I'll publish that new version. 👍